PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -2805,6 +2812,13 @@ int FindReplaceDlg::processAll(ProcessOperation op, const FindOption *opt, bool
 
 	int nbProcessed = processRange(op, findReplaceInfo, pFindersInfo, pOptions, colourStyleID);
 
+
+	// Turn ON the notifications after operations
+	(*_ppEditView)->execute(SCI_SETMODEVENTMASK, notifFlag);
+	if (op == ProcessReplaceAll && nbProcessed > 0) // All the notification of modification (SCN_MODIFIED) were removed during the operations, so we set modified status true here

```

Comment:
`NPPN_MODIFIEDREPLACEALL` could be added here for notifying the plugins "the current document has been changed after Replace All action".